README.md

```diff
@@ -18,63 +18,75 @@ issues in the [issue log](https://github.com/RadioAstronomySoftwareGroup/pyradio
+`pip install pyradiosky[healpix]`
+
+If you wish to manage dependencies manually read on.
+
+### Dependencies
+
+If you are using `conda` to manage your environment, you may wish to install the
+following packages before installing `pyradiosky`:
 
 Required:
 
-* numpy >= 1.15
-* scipy
-* astropy >= 4.0
+* astropy>=4.0
 * h5py
+* numpy>=1.15
+* scipy>1.0.1
 * pyuvdata
 * setuptools_scm
 
 Optional:
 
-* astropy-healpix (for working with beams in HEALPix formats)
+* astropy-healpix
+
+### Developer Installation
 
-The numpy and astropy versions are important, so make sure these are up to date.
+If you are developing `pyradiosky`, you will need to download and install the
+repository using `git clone https://github.com/RadioAstronomySoftwareGroup/pyradiosky.git`.
 
-We suggest using conda to install all the dependencies. If you want to install
-astropy-healpix, you'll need to add conda-forge as a channel
-(```conda config --add channels conda-forge```). Developers may wish to use
-the included `environment.yaml` file to create a new environment that will
-contain all the optional dependencies along with dependencies required for
-testing and development (```conda env create -f environment.yml```).
+If you use conda, you may wish to use a fresh environment, in which case you can
+use the included `environment.yaml` file to make a conda environment with all
+the extra dependencies required for testing/development as well as the
+standard ones using `conda env create -f environment.yml`. If you do not wish to
+make a fresh dedicated environment, you should verify that the environment you
+are using contains the packages listed in that file.
+Then do a developer install of pyradiosky using `pip install -e .` (or
+`pip install --no-deps -e .` if you do not want pip to install any missing
+requirements).
+
+If you do not use conda, after downloading the repository, install using
+`pip install -e .[dev]` to install all the extra dependencies required for
+testing/development as well as the standard ones.
+
+Finally, install the pre-commit hook using `pre-commit install` to help prevent
+committing code that does not meet our style guidelines.
 
-If you do not want to use conda, the packages are also available on PyPI.
-You can install the optional dependencies via pip by specifying an option
-when you install pyradiosky, as in ```pip install .[healpix]```
-which will install all the required packages for using the HEALPix functionality
-in pyradiosky. The options that can be passed in this way are:
-[healpix, dev]. The `healpix` enables various specific functionality
-while `dev` will install dependencies required for testing and development.
 
 ## Tests
 Uses the `pytest` package to execute test suite.
 From the source pyradiosky directory run ```pytest``` or ```python -m pytest```.
 
-# Maintainers
+## Maintainers
 pyuvdata is maintained by the RASG Managers, which currently include:
```

Comment:
Should this be "pyradiosky is maintained by the RASG Managers"?

Reply:
yes, thank you!